CHANGELOG.md

```diff
@@ -12,10 +12,13 @@ project adheres to [Semantic Versioning](http://semver.org/).
 ##### Added
 
 - Ability to configure force_delete for the worker group ASG (by @stefansedich)
+- write_aws_auth_config to input (by @yutachaos)
 
 ##### Changed
 
 - Change worker group ASG to use create_before_destroy (by @stefansedich)
+- Removed invalid action from worker_autoscaling iam policy
```

Comment:
I think this is a mistake? This change is not in this PR.

Reply:
Thank you fixed it.